### DISCUSSION TITLE
Data Accuracy and Data Updates using the API

### DISCUSSION THREAD

**Discussion by Akomandur** [meteostat/meteostat-python#50] [General] [answered]
Hi,

Thanks for creating Meteostats. I've found this much easier to use then any other solution out there.

I'm using the API to scrape hourly precipitation data for multiple GPS coordinates from one day ago. I have a script running at 3 AM to scrape this data. However, I've found that I get different precipitation values later in the day when running the same queries. Is this due to the API, or is it stemming from the weather data coming from the weather stations? I've tried searching for how often weather stations correct or update their observational values, but I didn't come up with much. 

Thanks!

**Comment by clampr** [comment, marked as answer]
Hi @Akomandur,

You're welcome. I'm glad you enjoy the UX 😄

What you describe is indeed expected behaviour. The reason is that Meteostat uses both model data and actual observations. It takes a few hours until observation data is coming in. Also, it might be that, at a later point of time, real time observations are replaced with quality-controlled data.

Perhaps, it would make sense to add some kind of FAQ to the docs to clarify these kinds of questions.

**Reply by Akomandur** [reply]
@clampr Thanks for the reply! Does meteostat use model data for data retrieved using the Stations class? I've been fetching data using the closest weather station since I thought those would be directly observed measurements. 

And I agree that FAQ for these types of questions might be helpful for other users like me.

**Comment by clampr** [comment]
Yes. By default, model data is used to fill up missing values in the time series. But you can disable model data by setting the `model` attribute to `False`.

Example:

```py
data = Hourly(station, start, end, model=False)
```

**Reply by Akomandur** [reply]
Great, thanks!
